common/proto/evm.token.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package messari.evm.token.v1;
+
+message Tokens {
+  repeated Token items = 1;
+}
+
+message Token {
+  string address = 1;
+  string name = 2;
+  string symbol = 3;
+  uint64 decimals = 4;
+  optional string total_supply = 5;
+}
+
+message Transfers {
+  repeated Transfer items = 1;
+}
+
+message Transfer {
+  string tx_hash = 1;
+  optional uint32 log_index = 2;
+  optional uint64 log_ordinal = 3;
+  Token token = 4;
+  string from = 5;
+  string to = 6;
+  string amount = 7; // BigInt, in token's native amount
+  optional string amount_usd = 8;
+  optional string reason = 9;
```

Comment:
is this needed?

Reply:
it was based off this: https://github.com/streamingfast/firehose-ethereum/blob/develop/proto/sf/ethereum/type/v2/type.proto#L411 would really only be used in eth-balance

Reply:
I'm not sure how interesting it is to include this. For most of our use cases so far, I don't think we've needed reason much if at all. Although we also never had access to this piece of this information. @yulesa what are your thoughts on this? Do you think there will be places where this piece of information can be useful?

Reply:
I have no idea what this is. I need context.

Reply:
I talked to @yulesa and "yes", but I also need to rethink how we want to show the eth balances, because there is more nuance.

ie: instead of thinking about it in the sense of balances at x block we should think about it in terms of balances and changes tied to calls